ci/bamboo/build-linux-debug.sh

```diff
@@ -5,11 +5,11 @@ set -o xtrace
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 # Run the common setup
-${DIR}/setup-dependencies.sh
+${DIR}/setup-dependencies-linux.sh
 apt-get install -y iwyu
 
 # Install nupic.core dependencies
-pip install \
+python -m pip install \
```

Comment:
The default installation location with `--user` option isn't in ${PATH}.
On Jul 11, 2016 5:38 PM, "Scott Purdy" <notifications@github.com> wrote:

> In ci/bamboo/build-linux-debug.sh
> <https://github.com/numenta/nupic.core/pull/1004#discussion_r70361640>:
>
> >  apt-get install -y iwyu
> >
> >  # Install nupic.core dependencies
> > -pip install \
> > +python -m pip install \
>
> Why isn't pip accessible?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/numenta/nupic.core/pull/1004/files/cd19cad4f4407b977195058c14864fa751a7c1b9#r70361640>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAT7gD8v8FKTk_zPDf1KPgCJDNtPCHr6ks5qUuHqgaJpZM4JJ3Ws>
> .
>